performance_manager/lib/postgres_schema.py

```diff
@@ -6,63 +6,63 @@
 SqlBase: Any = declarative_base()
 
 
-class VehiclePositionEvents(SqlBase):  # pylint: disable=too-few-public-methods
-    """Table for GTFS-RT Vehicle Position Events"""
+class VehicleEvents(SqlBase):  # pylint: disable=too-few-public-methods
+    """
+    Table that hold GTFS-RT Events
 
-    __tablename__ = "eventsVehiclePositions"
+    Store all of the information that identifies a trip and a stop. Then tie that
+    information with timestamp events when a vehicle begins moring to the stop and
+    when it arrived at the stop ()
+    """
+
+    __tablename__ = "vehicle_events"
 
     pk_id = sa.Column(sa.Integer, primary_key=True)
-    is_moving = sa.Column(sa.Boolean)
-    stop_sequence = sa.Column(sa.SmallInteger, nullable=True)
-    stop_id = sa.Column(sa.String(60), nullable=True)
-    timestamp_start = sa.Column(sa.Integer, nullable=False)
-    timestamp_end = sa.Column(sa.Integer, nullable=False)
-    direction_id = sa.Column(sa.SmallInteger, nullable=True)
-    route_id = sa.Column(sa.String(60), nullable=True)
-    start_date = sa.Column(sa.Integer, nullable=True)
-    start_time = sa.Column(sa.Integer, nullable=True)
+
+    # trip identifiers
+    direction_id = sa.Column(sa.Boolean, nullable=False)
+    route_id = sa.Column(sa.String(60), nullable=False)
+    start_date = sa.Column(sa.Integer, nullable=False)
+    start_time = sa.Column(sa.Integer, nullable=False)
     vehicle_id = sa.Column(sa.String(60), nullable=False)
-    hash = sa.Column(
-        sa.LargeBinary(16), nullable=False, index=True, unique=False
+
+    # stop identifiers
+    stop_sequence = sa.Column(sa.SmallInteger, nullable=False)
+    stop_id = sa.Column(sa.String(60), nullable=False)
+    parent_station = sa.Column(sa.String(60), nullable=False)
+
+    # hash of trip and stop identifiers
+    trip_stop_hash = sa.Column(
+        sa.LargeBinary(16), nullable=False, index=True, unique=True
     )
```

Comment:
I think it is worth add a `trip_hash` column at this point. It will likely be helpful for some of the metrics calculations, and OPMI might find use in it for their workflows. 

Dropping it in a future PR will be a lot easier than having to add it before we start re-loading tables.

Adding the column right before the insert operation in the `upload_to_database` function would be an easy drop in.